jenkins/Jenkinsfile.macos

```diff
@@ -96,196 +96,168 @@ pipeline {
       }
     }
 
-    stage("Build Matrix") {
+    stage('Build and Sign') {
 
       environment {
-        ENV = ""
+        AWS_ACCOUNT_ID = '749683154838'
+        KEYCHAIN_PASSPHRASE = credentials('ide-keychain-passphrase')
+        BUILD_AGENT_TEMP = "${env.HOME}/tmp"
+        MACOS_DEVELOPER_CERTIFICATE = credentials('MACOS_DEVELOPER_CERTIFICATE')
+        MACOS_DEVELOPER_CERTIFICATE_KEY = credentials('MACOS_DEVELOPER_CERTIFICATE_KEY')
       }
 
-      matrix {
+      steps {
+        script {
+          ENV = utils.getBuildEnv(!params.DAILY)
+        }
+        script {
+          // Create a keychain to hold the Developer ID certificate for signing
+          sh """
```

Comment:
Consider using single-quotes rather than double-quotes, so that expansion of things like `${FOO}` happens in the shell rather than through Groovy string interpolation.